docs/source/cn/comm.py

```diff
@@ -35,3 +35,44 @@
 
     """,
 )
+
+reset_docstr(
+    oneflow.comm.all_gather,
+    r"""
+    将整个进程组的张量收集到一个列表中。
+
+    参数：
+        - tensor_list (list[Tensor]): 输出列表。它应该包含收集到的正确尺寸的张量用于输出。
+        - tensor (Tensor): 从当前进程广播的张量。
+
+    样例：
+
+    .. code-block:: python
+
+        >>> # 我们有一个进程组，包含两个 rank。
+        >>> import oneflow as flow
+
+        >>> input = flow.tensor([[1, 2], [3, 4]], device="cuda") + flow.env.get_local_rank()
+        >>> # rank0 的输入
+        >>> input # doctest: +ONLY_CHECK_RANK_0
+        tensor([[1, 2],
+                [3, 4]], device='cuda:0', dtype=oneflow.int64)
```

Comment:
```suggestion
        # 我们有一个进程组，包含两个 rank。
        > import oneflow as flow

        > input = flow.tensor([[1, 2], [3, 4]], device="cuda") + flow.env.get_local_rank()
        # rank0 的输入
        > input
        tensor([[1, 2],
                [3, 4]], device='cuda:0', dtype=oneflow.int64)
```

我去看了下 oneflow 仓库的代码，这些 `# doctest: +ONLY_CHECK_RANK_0` 最后的本质作用，其实就是跳过 doctest 的检查。那么我们暂时也可以简化，把 docstring 不按照 doctest 的约定写，当作纯文本些，这样 doctest 就不会跑，跳过检查，就可以了。

@ChenhaoWang2000 @rainyq69

Reply:
收到